packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -65,26 +87,7 @@ const deployContract_NotWait = async (payload: {
   }
 };
 
-/**
- * Deploy a contract using the specified parameters.
```

Comment:
only the comments you added, not the previous ones, these ones serve as explanation

Reply:
@josephchimebuka

Reply:
oh okay i'll return it now